.travis.yml

```diff
@@ -6,10 +6,13 @@ scala:
 jdk:
   - oraclejdk8
 env:
-  # Setting this variable twice will cause the 'script' section to run twice with the respective env var invoked
-  - BUILD_TYPE=sbt
-  - BUILD_TYPE=centaurJes
-  - BUILD_TYPE=centaurLocal
+  global:
+    - CENTAUR_BRANCH=79dc5f9f0f6ca6aedb5b13358d47b0c9c4156d51
```

Comment:
It is a commit, but the variable is named after the the `TRAVIS_BRANCH` tag name used for the cromwell commit, that used to be plumbed through as `BRANCH` in the various centaur scripts.

In the future, one could create named branches (like `develop`), and then pin to the branch names instead of the current hashes.